books/management/commands/resync.py

```diff
@@ -71,7 +71,7 @@ def handle(self, *args, **options):
             if book:
                 if success:
                     counter['success'] = counter['success'] + 1
-                    self.stdout.write(self.style.SUCCESS(
+                    self.stdout.write(self.style.HTTP_REDIRECT(
```

Comment:
Yeah, some cleanup and convenience functions to avoid the ugly long lines would probably be in order. As for the information, personally I don't have a strong opinion about which specific fields to print, although I do find at least the "result" (imported/not imported due to X) quite informative.

It is also probably a good thing to have a "verbose" flag of sorts, that we can keep quite simple: if False, no info about the fields is printed; if True, all fields are printed. Eventually we can refine it and have different levels of verbosity, of course.

I'd probably lean towards leaving these changes to a further iteration, in the hopes of keeping this pull request a bit smaller, but they are definitely on my wish list!